### DISCUSSION TITLE
Unexpected error => Module not found: Can't resolve '../utils/logger'

### DISCUSSION THREAD

**Discussion by dimasday90** [nextauthjs/next-auth#4759] [Help]
Recently me and my teams is trying to deploy web application and it always do npm install and npm run build.. but suddenly got this error when build in deployment environment:

`Jun 23 18:43:17 ip-172-31-15-30 web: Failed to compile.
Jun 23 18:43:17 ip-172-31-15-30 web: ./node_modules/next-auth/react/index.js
Jun 23 18:43:17 ip-172-31-15-30 web: Module not found: Can't resolve '../utils/logger' in '/var/app/current/node_modules/next-auth/react'
Jun 23 18:43:17 ip-172-31-15-30 web: Import trace for requested module:
Jun 23 18:43:17 ip-172-31-15-30 web: ./pages/[namespace]/index.js
Jun 23 18:43:17 ip-172-31-15-30 web: ./node_modules/next-auth/react/index.js
Jun 23 18:43:17 ip-172-31-15-30 web: Module not found: Can't resolve '../utils/parse-url' in '/var/app/current/node_modules/next-auth/react'
Jun 23 18:43:17 ip-172-31-15-30 web: Import trace for requested module:
Jun 23 18:43:17 ip-172-31-15-30 web: ./pages/[namespace]/index.js`

Currently our web app is using NextJS (obviously) and the next-auth version in package.json is "^4.3.4".. So maybe there is something wrong in the latest package installed when we deploy..

Please help to resolve this.. Thank you very much

**Comment by jain-aditi** [comment]
Hi,

Am also facing the same issue.

```
./node_modules/next-auth/react/index.js:37:0
Module not found: Can't resolve '../utils/logger'
Import trace for requested module:
./pages/_app.js
```
Does anyone having idea on this?
